### PR TITLE
Update existed "che-e2e" image for using on CI 

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
Signed-off-by: Ihor Okhrimenko <iokhrime@redhat.com>

### What does this PR do?
Update existed "che-e2e" image for using on CI 

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/14346

### CODE REVIEW

**Comment by Ohrimenko1988** [comment]
ci-build

**Comment by dmytro-ndp** [comment]
Could you, please, push the image with fixup somewhere accessible form CRW Jenkins CI so that it was possible to check running of Happy path tests on fixed image on Jenkins CI?

**Comment by Ohrimenko1988** [comment]
@dmytro-ndp Sorry, I can't do it now. If you really need to ensure that everything works as expected you may follow instruction and build it by your own
<https://github.com/eclipse/che/blob/master/dockerfiles/e2e/README.md>

**Review by artaleks9 — APPROVED** [review]

**Comment by Ohrimenko1988** [comment]
@dmytro-ndp you may try image ```bujhtr5555/che-e2e```

**Comment by dmytro-ndp** [comment]
@Ohrimenko1988: thanks

**Review by ScrewTSW — APPROVED** [review]
LGTM

**Review by Katka92 — APPROVED** [review]

**Review by Katka92 — APPROVED** [review]
I can see you've added mounting of local tests. Have you thought about adding the mount for screenshots too?

**Comment by Ohrimenko1988** [comment]
@Katka92 Screenshots already included into mounted "e2e" folder

File: e2e/README.md [hunk 1]
```diff
@@ -23,12 +23,24 @@
 - Use environment variables which described in the **```'TestConstants.ts'```** file
 
 ## Docker launch
+
 - open terminal and go to the "e2e" directory
```
Comment (dmytro-ndp) [inline e2e/README.md:27]: @Ohrimenko1988: we might not need to go to the "e2e" directory if we don't mount it to docker container. WDYT?

Comment (Ohrimenko1988) [reply e2e/README.md:27]: Totally disagree, for launch the "npm run test-docker" command you exactly should be in the root of e2e project or bash will not recognize "npm" command

Comment (dmytro-ndp) [reply e2e/README.md:27]: Thanks for clarification.

**Pull request merged**